builder/builder.py

```diff
@@ -364,8 +367,13 @@ def publish(course_key: str) -> List[str]:
             raise Exception(f"Course directory not found for {course_key} - the course probably has not been built")
 
     symbolic_link(config)
+    errors = errors + publish_graders(config)
 
-    return errors + publish_graders(config)
+    # Remove temporary files that were created during renaming.
+    # This may block the execution for a while for courses with many files
+    rm_paths(tmpfiles)
```

Comment:
If removing files fails with IO errors, where are the exceptions caught? 
The view (`access.views.publish`) includes the try-except statement, which seems to be OK.
-> no changes needed